### PR TITLE
Better error message to help debug travis problem

### PR DESCRIPTION
**Pull request by nouiz**
You can make a pull request to Theano github repo in order to fix a bug or add new features.
Once a pull request is submitted, [Travis](https://travis-ci.org/) is launched to verify the pull request
given the current stack of unit-tests in Theano.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,12 +31,12 @@ addons:
 
 install:
 # In Python 3.3, we test the min version of NumPy and SciPy. In Python 2.7, we test more recent version.
-  - if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then conda create --yes -q -n pyenv mkl python=2.7 numpy=1.11.1 scipy=0.18 nose=1.3.0 pip flake8=2.3 six=1.9.0 pep8=1.6.2 pyflakes=0.8.1 sphinx mkl-service; fi
+  - if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then conda create --yes -q -n pyenv mkl python=2.7 numpy=1.9.1 scipy=0.14.0 nose=1.3.0 pip flake8=2.3 six=1.9.0 pep8=1.6.2 pyflakes=0.8.1 sphinx mkl-service libgfortran=1; fi
   - if [[ $TRAVIS_PYTHON_VERSION == '3.3' ]]; then conda create --yes -q -n pyenv mkl python=3.3 numpy=1.9.1 scipy=0.14.0 nose=1.3.4 pip flake8=2.3 six=1.9.0 pep8=1.6.2 pyflakes=0.8.1 sphinx mkl-service; fi
   - source activate pyenv
   - if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then pip install pydot; else pip install pydot-ng; fi
   - pip install . --no-deps
-  - pip install flake8-future-import nose-parameterized==0.5.0
```
Comment (abergeron) [inline .travis.yml]: ok for these

File: theano/tensor/tests/test_gc.py [hunk 2]
```diff
@@ -87,7 +89,17 @@ def c(fn):
...
+            for i in range(len_pre_f//100):
+                p1 = pre_f[i*100:(i+1)*100]
+                p2 = post_f[i*100:(i+1)*100]
+                if p1 != p2:
+                    print(i)
+                    print("p1")
+                    print(p1)
+                    print("p2")
+                    print(p2)
+        assert len_pre_f == len_post_f, (len_pre_f, len_post_f)
```
Comment (abergeron) [inline theano/tensor/tests/test_gc.py]: I'm less sure about this.  It doesn't really give any useful information to debug.

**Comment by abergeron** [comment]
I've been able to reproduce the problem on my machine, it just requires you run the full testsuite in the travis docker image.  If you run the test on its own it passes.

**Comment by nouiz** [comment]
travis seem to pass (1 part missing, but it isn't a section that was failing). I'll merge now to repair the master. We will see later to redo it.

**Pull request merged**